backend/usage_v2/models.py

```
@@ -7,23 +8,56 @@
+
+try:
+    from pluggable_apps.lookups.constants import (
+        CLOUD_LLM_USAGE_REASON_CHOICES,
+        CLOUD_REFERENCE_TYPE_CHOICES,
+    )
+
+    _LLM_USAGE_REASON_CHOICES.extend(CLOUD_LLM_USAGE_REASON_CHOICES)
+    _REFERENCE_TYPE_CHOICES.extend(CLOUD_REFERENCE_TYPE_CHOICES)
+except ImportError:
+    pass
+except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)